### ISSUE TITLE
Feature request: new automatic email

### ISSUE THREAD

**Issue by smilius42** [CCSDForge/episciences#121]
Dear colleagues,

at LMCS we sometimes have the problem that editors forget to start handling a paper which is newly assigned to them. Authors may then ask after some time (somemtimes more than half a year) about the status of their submission, but nothing was done, which is not good for the reputation of the journal.

We would like to request that a new automatic email is implemented. This email should be send when after a certain time (configurable) after an editor is assigned to a paper that editor did not invite any reviewers. The recipients should be the editor himself or the Editor in Chief or both (configurable).

We think this feature will greatly help mitigate the above problem.

Best regards,
Stefan 

p.s. Feature request copied from https://support.ccsd.cnrs.fr/SelfService/Display.html?id=51127

**Comment by clason** [comment]
Isn't that already possible with the "Automatic reminders" feature (under Journal settings->Mail->Automatic reminders->Add a new reminder: "Not enough reviewers: Editor's copy")?

**Comment by smilius42** [comment]
Not really, because this requires that you set a non-zero 'minimum ratings count'. Unfortunately, this minimum number of reviews is then enforced by the system globally through the whole process, i.e. also for all revised versions. However, it is common practise that editors only ask some of the reviewers in the subsequent reviewing rounds (or none if revisions were minor) and then proceed to notifying the authors. To make this workable one has to set the 'minimum rating count' to 0.  So actually, this feature request could also be formulated differently:

Make the 'minimum rating count' more flexible. That is, the 'minimum ratings count' should only apply for the first round of reviewing (or it could be configurable for how many rounds it should be applied).
